src/ctia/entity/event/obj_to_event.clj

```diff
@@ -22,10 +22,7 @@
   "Given the output of a `diff` between maps return a list
   of edit distance operation under the form of an Update map"
   [[diff-after diff-before _]]
-  (filter
-   (fn [{:keys [change]}]
-     (seq change))
-   (concat
+  (concat
```

Comment:
maybe `mapcat`?

Reply:
this `concat` is applied to the results of 2 `map`.